app/javascript/packages/address-search/webpack.config.cjs

```diff
@@ -0,0 +1,34 @@
+module.exports = /** @type {import('webpack').Configuration} */ ({
+  mode: 'production',
+  target: ['node'],
+  entry: {
+    index: './',
+  },
+  experiments: {
+    outputModule: true,
+  },
+  output: {
+    module: true,
+    chunkFormat: 'commonjs',
+    filename: '[name].js',
+    library: {
+      type: 'module',
+    },
+  },
+  resolve: {
+    extensions: ['.js', '.jsx', '.ts', '.tsx', '.mjs', '.cjs', '.mts', '.cts'],
+  },
+  externals: /^(?!(@18f\/identity-|\.))/,
```

Comment:
Externals would imply that the consumer would be responsible for providing these. But they're bundled in the output of this package, right? Do we need this option at all?

Reply:
Ohhhh, I missed the negation in the RegExp. You're right, that makes sense 👍